src/components/List/ListItem/ListItem.tsx

```diff
@@ -54,8 +54,10 @@ export const ListItem = forwardRef<HTMLLIElement, Props>(
             <li
                 {...liProps}
                 ref={ref}
-                role="presentation"
+                role="option"
```

Comment:
Without role it shows  **Non-interactive elements should not be assigned mouse or keyboard event listeners**   should we remove onClick={onClick} ?

Reply:
You are right. Role `list` is for static lists, while `listbox` is for selectable lists, which is what this component is about. Since we have `onKeyDown` and `tabindex` we should use `listbox` here, but not in the first render block.